### DISCUSSION TITLE
HOW TO BYPASS HEROKU BAN

### DISCUSSION THREAD

**Discussion by mewzax** [Vedza/NitroSniperGo#417] [General]
It looks like Heroku now bans this sniper.
To bypass it, just follow the steps:

1) Fork the repo
2) replace vedza name in your fork by your name to be less sus
3) Go to https://dashboard.heroku.com/new?template=https://github.com/[YOUR-GITHUB-NICKNAME]/NitroSniperGo/tree/heroku

Config, start sniper and its done !

**Comment by stzout** [comment]
I did this, but it only lasted for 1 hour before my account got suspended. Any other solutions?

**Comment by mewzax** [comment]
> hour

wow for me it works fine

**Comment by Mirlice** [comment]
@Mewzax How can I change the name of vedza to my pseudo ?

**Reply by mewzax** [reply]
replace vedza name by your name in all the code of the repo:
https://code.visualstudio.com/docs/editor/codebasics#_search-and-replace
